lib/travis/addons/config/notify.rb

```diff
@@ -49,6 +49,13 @@ def on_failure_for?(type)
             end
           end
 
+          def on_canceled_for?(type)
```

Comment:
Sorry, I haven't seen this PR before. Always better to ping me on Slack, too :)

I suppose we'd want an entry in the `DEFAULTS` hash for this, don't we? https://github.com/travis-ci/travis-hub/blob/d69767aaf17ac43fb7b4ea31f486282244943aba/lib/travis/addons/config/notify.rb#L13